Library/Formula/dlite.rb

```diff
@@ -0,0 +1,94 @@
+require "language/go"
+
+class Dlite < Formula
+  desc "The simplest way to use Docker on OSX"
+  homepage "https://github.com/nlf/dlite"
+  url "https://github.com/nlf/dlite/archive/1.1.1.tar.gz"
+  sha256 "e63b81be5fec5b3b55e3b364f4bc77cfd3ded3920a4cc124431a2680e7f46178"
+
+  # DLite depends on xhyve which only works on OSX versions 10.10 (Yosemite) or newer
+  depends_on :macos => :yosemite
+  depends_on "go" => :build
+
+  go_resource "github.com/TheNewNormal/libxhyve" do
+    url "https://github.com/TheNewNormal/libxhyve.git",
+    :revision => "0c94d077c947d92c08edcc359e15d2498edb70a5"
+  end
+
+  go_resource "github.com/briandowns/spinner" do
+    url "https://github.com/briandowns/spinner.git",
+    :revision => "63ad0cb315eae5b1ee029af25175c8bc0db690e7"
+  end
+
+  go_resource "github.com/fatih/color" do
+    url "https://github.com/fatih/color.git",
+    :revision => "9aae6aaa22315390f03959adca2c4d395b02fcef"
+  end
+
+  go_resource "github.com/hooklift/xhyve" do
+    url "https://github.com/hooklift/xhyve.git",
+    :revision => "219dc33c05fa5c7c0529751c029b4ac66844d67a"
+  end
+
+  go_resource "github.com/jessevdk/go-flags" do
+    url "https://github.com/jessevdk/go-flags.git",
+    :revision => "d91b7c5a5ce0b1d99d765ec3fb20ab590e52ddcb"
+  end
+
+  go_resource "github.com/johanneswuerbach/nfsexports" do
+    url "https://github.com/johanneswuerbach/nfsexports.git",
+    :revision => "a9068f3f0daa39616953aec11c3eb1209ebc4086"
+  end
+
+  go_resource "github.com/kardianos/osext" do
+    url "https://github.com/kardianos/osext.git",
+    :revision => "29ae4ffbc9a6fe9fb2bc5029050ce6996ea1d3bc"
+  end
+
+  go_resource "github.com/mattn/go-colorable" do
+    url "https://github.com/mattn/go-colorable.git",
+    :revision => "3dac7b4f76f6e17fb39b768b89e3783d16e237fe"
+  end
+
+  go_resource "github.com/mattn/go-isatty" do
+    url "https://github.com/mattn/go-isatty.git",
+    :revision => "56b76bdf51f7708750eac80fa38b952bb9f32639"
+  end
+
+  go_resource "github.com/nlf/dlite" do
+    url "https://github.com/nlf/dlite.git",
+    :revision => "ed934d49004885de49c67d9c6791f8b96b6107c7"
+  end
+
+  go_resource "github.com/satori/go.uuid" do
+    url "https://github.com/satori/go.uuid.git",
+    :revision => "d41af8bb6a7704f00bc3b7cba9355ae6a5a80048"
+  end
+
+  go_resource "github.com/shiena/ansicolor" do
+    url "https://github.com/shiena/ansicolor.git",
+    :revision => "a422bbe96644373c5753384a59d678f7d261ff10"
+  end
+
+  go_resource "github.com/ricochet2200/go-disk-usage" do
+    url "https://github.com/ricochet2200/go-disk-usage.git",
+    :revision => "f0d1b743428fc09792f93233d9412f89f88fa2ab"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "go", "build", "-o", "dlite"
+    bin.install "dlite"
```

Comment:
You can use `system "go", "build", "-o", "#{bin}/dlite"` and remove this line :wink: